packages/amplify_api/lib/method_channel_api.dart

```diff
@@ -26,8 +26,11 @@ import 'amplify_api.dart';
 
 const MethodChannel _channel = MethodChannel('com.amazonaws.amplify/api');
 
+AmplifyException nullMapFromMethodChannelException =
```

Comment:
Could you explain this change?

Reply:
The NS upgrade treats any information received from MethodChannels as nullable.

In our method_channel code we can either force cast this result, or throw an exception if the result is null. It's better to throw an exception instead of force cast, thus we throw "nullMapFromMethodChannelException" in case the returned result is actually null.